src/pbrt/base/medium.h

```diff
@@ -88,7 +88,6 @@ class Medium
 
     std::string ToString() const;
 
-    PBRT_CPU_GPU
```

Comment:
Why remove this?

Reply:
I thought that `isEmissive` is not supposed to run on GPU as it calls `DispatchCPU`: https://github.com/mmp/pbrt-v4/blob/master/src/pbrt/media.cpp#L153-L156

Reply:
Oops, yes, makes sense, good fix.